supplier_inventory_import_ubl/models/product.py

```diff
@@ -6,20 +6,54 @@
 from openerp.addons import decimal_precision as dp
 
 
-class ProductProduct(models.Model):
-    _inherit = "product.product"
+HELP = "Contextual info on last supplier stock updated with UBL import"
+
+
+class ProductSupplierInventoryUblMixin(models.AbstractModel):
+    _name = "product.supplier.inventory.ubl.mixin"
```

Comment:
I don't really see the point in doing a mixin here since the fields are overriden for product template.

Reply:
I can remove yes